### DISCUSSION TITLE
draw zigzag

### DISCUSSION THREAD

**Discussion by Masoud1365** [highfestiva/finplot#398] [General]
how can i draw zigzag using several line in finplot ? (in ohlc mode)

**Comment by highfestiva** [comment]
One possibility is to add several lines that attach to one another. Does that fit your needs? If so, see `examples/line.py`.

**Reply by Masoud1365** [reply]
thanks for your answer
can you show me an example to display several lines?
my code looks like this but this code does not draw any line
`df = df[['Open', 'Close', 'High', 'Low']]
      
        fplt.candlestick_ochl(df)
     
        line = fplt.add_line((df.iloc[30].High, 4.4), (df.iloc[49].Low, 4.6),
                             color='#000000', interactive=False)
        fplt.show()`

**Reply by highfestiva** [reply]
Just copy-paste the `add_line()` call with new x and y coordinates.

**Reply by Masoud1365** [reply]
did you mean codes should be like this ?

`df = df[['Open', 'Close', 'High', 'Low']]

    fplt.candlestick_ochl(df)
 
    fplt.add_line((df.iloc[10].High, 4.4), (df.iloc[15].Low, 4.6),color='#000000', interactive=False)
    fplt.add_line((df.iloc[15].Low, 4.4), (df.iloc[25].High, 4.6),color='#000000', interactive=False)
    fplt.add_line((df.iloc[25].High, 4.4), (df.iloc[35].Low, 4.6),color='#000000', interactive=False)
    fplt.show()`

**Reply by highfestiva** [reply]
Sure, also learn how to program is a good start, if that's what you're going to do.
